src/universal/modules/teamDashboard/containers/TeamColumns/TeamColumnsContainer.js

```diff
@@ -30,9 +31,7 @@ class TeamColumnsContainer extends Component {
   }
 
   getProjectById = (projectId) =>
-    this.props.viewer.projects.edges
-      .map(({ node }) => node)
-      .find(({ id }) => projectId === id);
+    getProjectById(this.props.viewer.projects, projectId);
```

Comment:
this.getProjectById calling getProjectById is confusing, but i'm cool to ship now fix later

Reply:
Just emulating currying.

Reply:
maybe this is a good place for some of that newage currying all the kids are doing ;-)

Reply:
Well, we either write the function as explicitly curried, or we mess with `Function.prototype` to make everything automatically curried...